src/plugins/discover/public/application/main/components/sidebar/discover_index_pattern_management.tsx

```diff
@@ -33,10 +33,9 @@ export interface DiscoverIndexPatternManagementProps {
 }
 
 export function DiscoverIndexPatternManagement(props: DiscoverIndexPatternManagementProps) {
-  const { indexPatternFieldEditor, core } = props.services;
+  const { dataViewFieldEditor, core } = props.services;
   const { useNewFieldsApi, selectedIndexPattern, editField } = props;
-  const indexPatternFieldEditPermission =
-    indexPatternFieldEditor?.userPermissions.editIndexPattern();
+  const indexPatternFieldEditPermission = dataViewFieldEditor?.userPermissions.editIndexPattern();
```

Comment:
should we already start to rename `indexPatternFieldEditPermission` to `dataViewFieldEditPermission`?

Reply:
I am not sure, since the method on the `dataFieldEditor` is still called `editIndexPattern`. I am divided on this one.

Reply:
Same as the file name, can it be `discover_data_view_management` already?

Reply:
if we intend to clean it up, I think the file should also be renamed, however this can also be done in multiple PRs, since the title of this PR is very specific